optimum/intel/openvino/utils.py

```diff
@@ -22,3 +22,6 @@
 ONNX_ENCODER_NAME = "encoder_model.onnx"
 ONNX_DECODER_NAME = "decoder_model.onnx"
 ONNX_DECODER_WITH_PAST_NAME = "decoder_with_past_model.onnx"
+
+MAX_ONNX_OPSET_2022_2_0 = 10
+MAX_ONNX_OPSET = 16
```

Comment:
MAX_ONNX_OPSET also depends on the PyTorch version being used. PyTorch 1.12 claims opset 13 support: https://github.com/pytorch/pytorch/releases/tag/v1.12.0. I would suggest changing this to 13 as well because this is what OpenVINO claims as well.